assets/scripts/configure-microk8s-windows.ps1

```diff
@@ -28,6 +30,18 @@ function cleanseString($inputSt) {
 
     return $finalString
 }
+function setSafeDirectory($repoPath) {
+    $distro = cleanseString(wsl bash -c 'echo $WSL_DISTRO_NAME');
+    $distro = $distro.ToString().Replace("`r`n","");
```

Comment:
Have you tried using string Trim() method here?
https://devblogs.microsoft.com/scripting/trim-your-strings-with-powershell/